test/listen.test.js

```diff
@@ -8,7 +8,7 @@ const listen = require('../listen')
 // TODO this is... not listen any more
 // we may need to rename this
 
-test('listen.addMember', t => {
+test.skip('listen.addMember', t => {
```

Comment:
TODO (undo skip)

Reply:
the addMember listener is removed though. so just remove the test too? although we'll see after i've rewritten stuff again